docs/html/tabs/tabs.html

```diff
@@ -0,0 +1,29 @@
+<div class="ray-tabs">
```

Comment:
consider adding the proper aria attrs to the example

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role

so when people copy/paste they can just edit them in super easily

Reply:
https://material.io/develop/web/components/tabs/tab-bar/ use material as a reference